packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1198,10 +1198,10 @@ contract VotingV2 is
     }
 
     function _requireRegisteredContract() private view {
-        if (migratedAddress != address(0)) require(msg.sender == migratedAddress);
-        else {
-            Registry registry = Registry(finder.getImplementationAddress(OracleInterfaces.Registry));
-            require(registry.isContractRegistered(msg.sender), "Caller must be registered");
-        }
+        Registry registry = Registry(finder.getImplementationAddress(OracleInterfaces.Registry));
```

Comment:
unrelated to your change this adds a pretty large fixed gas cost to the DVM for all price requests. in fact we have similar cross-chain calls in other places (like [here](https://github.com/UMAprotocol/protocol/blob/07f1f1806b7b786e7922ab994cd8258245b4ff4b/packages/core/contracts/oracle/implementation/VotingV2.sol#L342)). this adds pretty considerable fixed cost to making price requests.

we should consider adding gas optimizations as we've done in other UMA contracts by storing instances of these contracts and providing a "sync" method to pull the latest from the finder periodically.